src/UsersView/Users.tsx

```diff
@@ -0,0 +1,72 @@
+
+import { Component } from "react"
+import * as React from "react"
+import { Text, View, ScrollView, StyleSheet, Button } from "react-native"
+import { styles } from './Users.style'
+import { UserModel } from './UserModel';
+
+interface ComponentProps {
+    userCollection: UserModel[]
+    userSelectCollection: UserModel[]
+    onSelectedEvent: (selectedUser: UserModel[]) => void
+}
+
+interface SelectedUser {
+    [key: number]: UserModel
+}
+
+interface ComponentState {
+    selectedUser: SelectedUser
+}
+
+export class Users extends Component<ComponentProps, ComponentState>
+{
+    constructor(props: any){
+        super(props)
+        this.state = {
+            selectedUser: {}
+        }
+    }
+
+    renderButtons() {
```

Comment:
its better to use fat arrow function
```
renderButtons = () => {}
```

to guarantee that `this` refer to UsersComponent